### DISCUSSION TITLE
Issue with Neighbor Orientation Comparison (Bad Data) on 2D EBSD Data

### DISCUSSION THREAD

**Discussion by maliesha95** [BlueQuartzSoftware/DREAM3DNX-Issues#13] [Q&A]
Hi,

I need to extract the grain statistics using EBSD point data. So I'm trying to use the "Neighbor Orientation Comparison (Bad Data)" filter to clean up my 2D EBSD data before extracting the statsitics. However, this filter, along with many others such as "Segment Features," requires cell data called "Quaternions," which I don't have available for my EBSD point data. I believe this type of data is not applicable to 2D EBSD data. How can I use the Neighbor Orientation Comparison (Bad Data) filter and other filters that require "Quaternions" data on my 2D EBSD data?

Thanks

**Comment by StopkaKris** [comment]
This question was posted and answered on the DREAM.3D google group, so this post can be closed/resolved.

https://groups.google.com/g/dream3d-users/c/JCL64jjwnv8
